src/stactools/cli/commands/info.py

```diff
@@ -1,16 +1,34 @@
 import click
 import pystac
+from pystac import Catalog, STACObject
 
 
-def print_info(catalog_path: str, skip_items: bool = False) -> None:
+def print_info(
+    catalog_path: str, skip_items: bool = False, print_progress: bool = False
+) -> None:
     cat_count, col_count, item_count = 0, 0, 0
     cat_ext, col_ext, item_ext = set([]), set([]), set([])
 
     cat = pystac.read_file(catalog_path)
-    if not isinstance(cat, pystac.Catalog):
+    if not isinstance(cat, Catalog):
         print("{} is not a catalog".format(catalog_path))
         return
 
+    cat_id_info = "Catalog ID: {}".format(cat.id)
+
+    max_line_length = 0
+
+    def progress_print(root: STACObject) -> None:
+        nonlocal max_line_length
+        line = (
+            f"\rCatalogs={cat_count}, Collections={col_count}, "
+            "Items={item_count}: Reading {root}"
```

Comment:
Need an `f` in front of `"Items={item_count}: Reading {root}"` in order to see the variable values:

`f"Items={item_count}: Reading {root}"`

Looks good otherwise!

Reply:
good catch thanks!